app/src/main/res/layout/navigation_header_bill.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.navigation_header_bill` appears to be unused
 Unused resources make applications larger and slow down builds.